queries.sql

```diff
@@ -157,4 +157,13 @@ SELECT species.name, COUNT(*)
   WHERE vets.name = 'Maisy Smith'
   GROUP BY species.name
   ORDER BY COUNT(*) DESC
-  LIMIT 1;
\ No newline at end of file
+  LIMIT 1;
+
+SELECT COUNT(*) FROM visits where animal_id = 4;
+CREATE INDEX visits_animal_id ON visits(animal_id);
+
+SELECT * FROM visits where vet_id = 2;
+CREATE INDEX visits_vet_id ON visits(vet_id);
```

Comment:
- [ ] It would be better if our query results outputs the `index scan` as seen in the two other queries. However, for our query `vet_id = 2`, the query results outputs `seq scan` instead of the `index scan`. A quick fix is to remove the index and add it again
![Screenshot from 2023-01-31 06-38-17](https://user-images.githubusercontent.com/46388113/215684135-736e4725-0a06-4947-9a4a-4d0f5f478218.png)